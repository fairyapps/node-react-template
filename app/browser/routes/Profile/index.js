import Helmet from "react-helmet";
import React, { Component } from "react";
import { Container, Column, Row } from "../../components/Grid";
import Discover from "./components/Discover";
import About from "./components/About";

const Profile = props =>
  <Container>
    <Helmet title="Profile" />
    <Discover />
    <About />
  </Container>;

export default Profile;
