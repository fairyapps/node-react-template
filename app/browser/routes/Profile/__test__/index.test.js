import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Profile from "../index";

test("Profile renders correct snapshot", () => {
  const component = renderer.create(
    <Profile match={{ params: { id: "1" } }} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
