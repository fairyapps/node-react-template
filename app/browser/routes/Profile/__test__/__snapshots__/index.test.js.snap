// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile renders correct snapshot 1`] = `
.kjJANk {
  max-width: 100%;
  margin: 0 auto;
  padding-right: 10px;
  padding-left: 10px;
  margin-top: auto;
}

@media (min-width: 768px) {
  .kjJANk {
    max-width: 750px;
  }
}

@media (min-width: 992px) {
  .kjJANk {
    max-width: 970px;
  }
}

@media (min-width: 1200px) {
  .kjJANk {
    max-width: 1170px;
  }
}

<div
  className="sc-bdVaJa kjJANk"
>
  <h1>
    Avatar Here
  </h1>
  <h1>
    About Here
  </h1>
</div>
`;
